test/schema/export_test.js

```diff
@@ -21,14 +24,27 @@ describe('#exportToSchemasUniqueCases()', () => {
     let schemas = exportToSchemas([ns]);
     expect(schemas).to.have.length(16);
     expect(schemas).to.eql(expectedSchemas);
+    expect(validateSchema(schemas));
   });
 });
 
 function exportNamespaces(...namespace) {
   let schemas = exportToSchemas(namespace);
+  expect(validateSchema(schemas));
   return schemas;
 }
 
 function importFixture(name, ext='.json') {
   return JSON.parse(fs.readFileSync(`${__dirname}/fixtures/${name}${ext}`, 'utf8'));
 }
+
+function validateSchema(schema) {
+    const v = new validator();
+    const extendedSchema = importSchema();
+    return v.validate(schema, extendedSchema);
```

Comment:
So, and interesting thing I noticed in the readme: When oneOf or anyOf validations fail, errors that caused any of the sub-schemas referenced therein to fail are not reported, unless options.nestedErrors is truthy. This option may be useful when troubleshooting validation errors in complex schemas. 

I'm going to try to play with the error options to see if I can catch the underlying errors. If not, we may be better served to plug in a new module to get our desired results. 

At least the other logic is plugged in in the appropriate spot, so subbing one call to another is the only item left for this.